packages/Button/src/Button.styles.js

```diff
@@ -46,11 +68,18 @@ const disabledTextStyles = css`
 `;
 
 const activeStyles = css`
-  border-color: ${tokens.highlight.active.noBorder.borderColor};
-  box-shadow: ${tokens.highlight.active.noBorder.boxShadow};
+  ${stylers.focusRing.bordered()}
+`;
 
-  &:hover:not([disabled]):not([aria-disabled="true"]) {
-    border-color: ${tokens.highlight.active.noBorder.borderColor};
+const mouseStyles = css`
+  [data-whatinput="mouse"] &:not([data-has-forced-focus="true"]):focus {
```

Comment:
perhaps this could be setup as a mixin/definition in helpers? `[data-whatinput="mouse"] &:not([data-has-forced-focus="true"]):focus` rather than having to write this out every time....

Reply:
It's only used once in Button and once in RawButton now, and I think moving it to a common `helpers` helper in a third package is a bit harder to follow.  Prefer to follow the WET rule here vs the DRY one.  Or Martin Fowler's "Rule of Three", if you prefer:

https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)